tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -78,11 +78,11 @@ def __init__(self, context: str, sandbox: Optional[str] = None):
 class SerializationOptions(Dict[str, Any]):
     def __init__(
             self,
-            max_dom_depth: Optional[int] = None,
+            max_dom_depth: Optional[Union[int, None]] = 0,
             max_object_depth: Optional[int] = None,
             include_shadow_tree: Optional[str] = None
     ):
-        if max_dom_depth is not None:
+        if max_dom_depth != 0:
```

Comment:
We should not pass this if condition if the value is `None`. Then the optional flag comes into play, and the field should not be added.